TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -155,26 +157,18 @@ public function deleteAction(\TYPO3\Media\Domain\Model\Asset $asset)
     public function relatedNodesAction(Asset $asset)
     {
         $userWorkspace = $this->userService->getPersonalWorkspace();
+        $usageReferences = $this->assetService->getUsageReferences($asset);
         $relatedNodes = [];
-        foreach ($this->getRelatedNodes($asset) as $relatedNodeData) {
-            $accessible = $this->domainUserService->currentUserCanReadWorkspace($relatedNodeData->getWorkspace());
-            if ($accessible) {
-                $context = $this->createContextMatchingNodeData($relatedNodeData);
-            } else {
-                $context = $this->createContentContext($userWorkspace->getName());
-            }
-            $site = $context->getCurrentSite();
-            $node = $this->nodeFactory->createFromNodeData($relatedNodeData, $context);
-            $flowQuery = new FlowQuery([$node]);
-            /** @var Node $documentNode */
-            $documentNode = $flowQuery->closest('[instanceof TYPO3.Neos:Document]')->get(0);
-            $relatedNodes[$site->getNodeName()]['site'] = $site;
-            $relatedNodes[$site->getNodeName()]['documentNodes'][$documentNode->getIdentifier()]['node'] = $documentNode;
-            $relatedNodes[$site->getNodeName()]['documentNodes'][$documentNode->getIdentifier()]['nodes'][] = [
-                'node' => $node,
-                'nodeData' => $relatedNodeData,
-                'contextDocumentNode' => $documentNode,
-                'accessible' => $accessible
+
+        /** @var AssetUsageInNodeProperties $usage */
+        foreach ($usageReferences as $usage) {
+            $relatedNodes[$usage->getSite()->getNodeName()]['site'] = $usage->getSite();
```

Comment:
This results in a _Call to undefined method TYPO3\Neos\Domain\Strategy\AssetUsageInNodePropertiesStrategy::getSite()_ when I click on the _usages_ button in an asset's single view